zebra/zebra_router.h

```diff
@@ -110,8 +110,15 @@ struct zebra_router {
 	 * The EVPN instance, if any
 	 */
 	struct zebra_vrf *evpn_vrf;
+
+	/*
+	 * Time for when we sweep the rib from old routes
+	 */
+	time_t startup;
```

Comment:
startup_time ?

Reply:
doesn't the time_t already imply that?

Reply:
It implies at the declaration. But this variable is referred as zrouter.startup elsewhere. It doesn't imply the meaning there.